Jenkinsfile

```diff
@@ -56,22 +56,26 @@ def buildEdirectory(){
   sh 'wget http://zugpronas:5000/fbsharing/ofPMJOCK -O /tmp/edirectory.tar.gz'
   sh 'tar -xvf /tmp/edirectory.tar.gz -C /tmp'
   sh 'docker load --input /tmp/edir920.tar'
-  def image = docker.image("edirectory:9.2.0")
+  def name = "edirectory:9.2.0"
+  def image = docker.image(name)
   docker.withRegistry('https://docker-registry.ivyteam.io', 'docker-registry.ivyteam.io') {
     if (env.BRANCH_NAME == 'master') {
       image.push()
+      sh "docker image rm ${name}"
     }
   }
 }
 
 def build(def directory) {
   def tag = directory.replace("/", "-")
+  def name = "axonivy/build-container:${tag}"
   echo "Building container tag $tag in directory $directory"
   dir (directory) {
     docker.withRegistry('', 'docker.io') {
-      def image = docker.build("axonivy/build-container:${tag}")
+      def image = docker.build(name)
       if (env.BRANCH_NAME == 'master') {
         image.push()
+        sh "docker image rm ${name}"
```

Comment:
you can remove it - even if you are not on master branch

Reply:
I thought then is the whole work lost, is that clever?

Reply:
I never connected to a jenkins node and tried out the image there. 
I locally develop the image and build it. Here I only build it to ensure that the image can be built.

Reply:
Ok